bundle/artifacts/upload.go

```diff
@@ -29,7 +29,7 @@ func (m *cleanUp) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics
 	// We intentionally ignore the error because it is not critical to the deployment
 	err := client.Delete(ctx, ".", filer.DeleteRecursively)
 	if err != nil {
-		log.Errorf(ctx, "failed to delete %s: %v", uploadPath, err)
+		log.Debugf(ctx, "failed to delete %s: %v", uploadPath, err)
```

Comment:
We should evaluate the error classes in this condition and ignore the predictable errors (ie. the path doesn't exist) and return an error on real errors (like below).